lib/pubnub/validators/revoke_token.rb

```diff
@@ -0,0 +1,31 @@
+# Toplevel Pubnub module.
+module Pubnub
+  # Validator module that holds all validators modules
+  module Validator
+    # Validator for Grant event
+    module RevokeToken
+      include CommonValidator
+
+      def validate!
+        validate_keys!
+        validate_token!
+      end
+
+      private
+
+      def validate_keys!
+        raise(
+          ArgumentError.new(object: self, message: ':subscribe_key is required for grant token event.'),
+          ':subscribe_key is required for grant token event.'
+        ) if @subscribe_key.nil? || @subscribe_key.empty?
+      end
+
+      def validate_token!
+        raise(
+          ArgumentError.new(object: self, message: ':ttl has to be kind of Integer for grant token event.'),
+          ':ttl has to be kind of Integer for grant token event.'
```

Comment:
@kleewho This error message looks misleading. We are checking on presence of `token` in passed parameters, but report about missing `ttl`.

Reply:
sure. I'll fix it

Reply:
Of course it's a copy paste error 😢